setup.cfg

```diff
@@ -102,7 +102,7 @@ doc =
     jupyter
     jupyter_client < 8.0.0
     tabulate
-    shinylive @ git+https://github.com/posit-dev/py-shinylive.git@main
+    shinylive @ git+https://github.com/posit-dev/py-shinylive.git
```

Comment:
If we release `py-shinylive` before `shiny`, we'd want to set this line to the newly released version before releasing `shiny`.